src/main/resources/db/migration/V1.11__add_location_reference.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS location_reference (
```

Comment:
Currently your PR failed to be deployed in Jenkins. It is likely to be caused by this change. Think we need to discuss this